terraform/environments/mlra/codebuild.tf

```diff
@@ -1,6 +1,7 @@
-# TODO change the module name to codebuild when rebuilding
 
 module "mlra-selenium" {
+  #checkov:skip=CKV_AWS_316:TODO Will be addressed as part of https://dsdmoj.atlassian.net/browse/LASB-3390
+  #checkov:skip=CKV_AWS_314:TODO Will be addressed as part of https://dsdmoj.atlassian.net/browse/LASB-3390
```

Comment:
Any reason for these duplicate comments added?

Reply:
The thinking was that when these checks are addressed, each check skip will be removed and addressed one by one.
The duplication is to ensure that any/all remaining `checkov:skip` retain a clear link to the Jira.
I can remove the duplicated ones (on neighbouring lines) if you think it's overkill with the duplication?

Reply:
I'm OK to leave as is then, thanks.